props/prop_base.h

```diff
@@ -207,24 +207,24 @@ class PropBase : CommandParser, Looper, protected SaberBase {
       current_preset_.variation = SaberBase::GetCurrentVariation();
       current_preset_.Save();
     }
-#endif	
+#endif
```

Comment:
None of the changes in this file seem relevant to the button changes.

Reply:
I think my editor did a bunch of whitespace garbage.  I'll try to correct it.

Reply:
fixed whitespace on prop_base.h